iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/datastructure/TVList.java

```diff
@@ -281,14 +430,20 @@ protected void cloneAs(TVList cloneList) {
       cloneList.timestamps.add(cloneTime(timestampArray));
     }
     cloneList.rowCount = rowCount;
+    cloneList.seqRowCount = seqRowCount;
     cloneList.sorted = sorted;
     cloneList.maxTime = maxTime;
+    cloneList.minTime = minTime;
   }
 
   public void clear() {
     rowCount = 0;
+    seqRowCount = 0;
     sorted = true;
     maxTime = Long.MIN_VALUE;
+    minTime = Long.MAX_VALUE;
+    queryContextList.clear();
+    ownerQuery = null;
```

Comment:
How about indices and bitmaps?

Reply:
indices and bitmaps are cleared in clearValue.
```
  void clearValue() {
    if (values != null) {
      for (boolean[] dataArray : values) {
        PrimitiveArrayManager.release(dataArray);
      }
      values.clear();
    }
    clearSlicesAndBitMap();
  }
```
AlignedTVList already clear indices and bitmaps in clearValue method.

Reply:
Done